pkg/auth/dependency/sso/auth_handler_html.go

```diff
@@ -21,28 +19,98 @@ func (i *AuthHandlerHTMLProvider) HTML() (out string, err error) {
 <!DOCTYPE html>
 <html>
 <head>
-<meta name=viewport content="width=device-width, initial-scale=1">
-<meta http-equiv="Content-Type" content="text/html; charset=utf-8">
-<script type="text/javascript" src="{{ js_sdk_cdn_url }}"></script>
 <script type="text/javascript">
-skygear.pubsub.autoPubsub = false;
-skygear.config({
-	'endPoint': '{{ api_endpoint }}',
-	'apiKey': '-'
-}).then(function(container) {
-	skygear.auth.oauthHandler();
-}, function(err) {
-	console.error(err);
-});
+function cookieJarGet(name) {
+	var jar = [];
+	var pairs = document.cookie.split("; ");
+	var i = 0;
+	for (; i < pairs.length; ++i) {
+		var parts = pairs[i].split("=");
+		var key = parts[0];
+		var value = parts.slice(1).join("=");
+		jar.push([key, value]);
+	}
+	for (var i = 0; i < jar.length; ++i) {
+		if (jar[i][0] === name) {
+			return jar[i][1];
+		}
+	}
+}
+
+function cookieJarDelete(name) {
+	document.cookie = name + "=;expires=Thu, 01 Jan 1970 00:00:01 GMT;";
+}
+
+function StringStartsWith(s, search) {
+	return s.substring(0, search.length) === search;
+}
+
+function validateCallbackURL(callbackURL, authorizedURLs) {
```

Comment:
This logic does not seems matches what server does?

Reply:
This is the original code.

Reply:
Do we fix it to match server logic now or later?

Reply:
I am going to make the two implementation the same.